### ISSUE TITLE
method does not override or implement a method from a supertype

### ISSUE THREAD

**Issue by refikanissa** [SimenCodes/react-native-imei#2]
When I built my project after linking react-native-imei, I got an error : method does not override or implement a method from a supertype
    @Override
    ^
1 error
:react-native-imei:compileReleaseJavaWithJavac FAILED

**Comment by asorg74** [comment]
me too, the same error msg when I tried to run-android

**Comment by koswarabilly** [comment]
it has been so long but i still get the same error

**Comment by arcollector** [comment]
update the npm version please my friend

**Comment by SimenCodes** [comment]
Sorry everyone, have been busy lately and didn't realized there were actually people using this library until recently (this was originally something I coded for a project at work that we eventually didn't need, so I have to admit that I kind of forgot about the whole thing too).

Don't want to merge any of the PR's because simply removing the override annotation isn't the right thing to do here (the whole method should have been removed). Pushing a new version (0.1.1) to NPM in a few moments, let me know if you run into any problems with it.
